assets/app/scripts/services/applicationGenerator.js

```diff
@@ -122,7 +122,7 @@ angular.module("openshiftConsole")
       templateLabels.deploymentconfig = input.name;
 
       var deploymentConfig = {
-        apiVersion: osApiVersion,
+        apiVersion: oApiVersion,
         kind: "DeploymentConfig",
```

Comment:
actually, check in views as well... not sure if we're switching in trigger type for deployment config and/or build config